src/Agent/NewRelic/Agent/Core/DataTransport/DataStreamingService.cs

```diff
@@ -677,6 +678,7 @@ private bool GetRequestStreamWithRetry(int consumerId, CancellationToken cancell
                 }
                 catch (Exception ex)
                 {
+                    // **IMPORTANT** None of this error handling logic is hit for grpc-dotnet since the we fake out creating/testing streams
```

Comment:
```suggestion
                    // **IMPORTANT** None of this error handling logic is hit for grpc-dotnet since we fake out creating/testing streams
```